runelite-client/src/main/java/net/runelite/client/plugins/fishing/FishingPlugin.java

```diff
@@ -210,4 +218,44 @@ public void checkSpots(GameTick event)
 		Arrays.sort(spots, Comparator.comparing(npc -> -1 * npc.getLocalLocation().distanceTo(cameraPoint)));
 		fishingSpots = spots;
 	}
+
+	@Subscribe
+	public void onGameTick(GameTick event)
+	{
+		for (NPC npc : fishingSpots)
```

Comment:
You should probably add if (!config.showMinnowOverlay()) at the start of the method as early return